src/ukify/test/test_ukify.py

```diff
@@ -601,7 +601,7 @@ def test_pcr_signing(kernel_initrd, tmpdir):
     priv = unbase64(ourdir / 'example.tpm2-pcr-private.pem.base64')
 
     output = f'{tmpdir}/signed.efi'
-    opts = ukify.parse_args([
+    opts = [
```

Comment:
I'd like to invert naming, like my PR, this should be `args`, and then `opts = ukify.parse_args()`, for consistency with other places.